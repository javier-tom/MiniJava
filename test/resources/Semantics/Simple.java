class SimpleMain {
    public static void main(String[] args) {
        System.out.println(new Simple().run());
    }
}

class Simple {
    public int run() {
        return 17;
    }
}
