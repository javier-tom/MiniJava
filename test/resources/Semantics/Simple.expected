class Simple:
  super: null
  fields:
  method run:
    params: 
    returns: int
    locals:
